src/main/java/frc/robot/Robot.java

```diff
@@ -108,9 +102,13 @@ public void robotInit() {
         // chooser.addOption("Mid Note + 2 Center Line", "Mid Note + 2 Center Line");
         // (Impossible)
         chooser.addOption("Start Mid, Close left, Close Mid, Far Left", "Optimized 4 Note Auto");
+        chooser.addOption("Start Source, Close Source, Far Source, FM Source", "CSFSFSM");
+        chooser.addOption("3 Center Line", "3 Center Line");
         // 5 Note Auto
-        chooser.addOption("5 Note Auto", "Optimized 5 Note Auto");
-        chooser.addOption("Faster 5 Note Auto", "Event Marker 5 Note");
+        chooser.addOption("Left 5 Note", "Left 5 Note");
+        chooser.addOption("5 Note, all close, far middle", "All Close + FM");
+        chooser.addOption("Center 5 Note", "Center 5 Note");
+        chooser.addOption("Original 5 Note", "OffbrandEventMarkers");
```

Comment:
Give this a descriptive name too probably. I know people know what this is, but still